docs/cloud-init.yml

```diff
@@ -94,6 +120,12 @@ write_files:
 
 runcmd:
 
+# configure "global" variables
+  - export BH_DOMAIN="demo.bhi.ma"
+  - export NODE_VERSION="12"
```

Comment:
Why do we choose the version 12 ? if the reason is the latest LTS it can be useful to have a comment which mention that

Reply:
Yeah, I should leave a comment about it.  I'll do that.